greenCan/accounts/tests/test_views.py

```diff
@@ -119,7 +119,25 @@ def test_post_method_with_invalid_data_4(self):
         response = self.client.post(self.url, data, follow=True)
         message = list(response.context.get("messages"))[0]
         self.assertEquals(message.tags, "error")
-        self.assertEquals(message.message, "Password is required")
+        # self.assertEquals(message.message, "Password is required")
```

Comment:
Why did you remove this line from the test?

Reply:
I thought it was necessary to test if it was sending an error msg in case if the password is not provided.